### PR TITLE
continuation of #374

### PR DESCRIPTION
**Pull request by ananthakumaran**
#374

### CODE REVIEW

**Comment by ananthakumaran** [comment]
@samidarko @LukevanTricht would appreciate if you guys could review the api.

**Comment by LukevanTricht** [comment]
I like it, clean, simple, does the job it needs to do!

File: lib/exq/mock.ex [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    }
+  end
+
+  defp to_job([_pid, queue, _time_or_offset, worker, args, _options]) do
+    %Job{
+      jid: UUID.uuid4(),
+      queue: queue,
+      class: worker,
+      args: args,
+      enqueued_at: DateTime.utc_now()
```
Comment (sergey-chechaev) [inline lib/exq/mock.ex:160]: Why did you decide not to use time_or_offset and set DateTime.utc_now() for Mock?

Comment (ananthakumaran) [reply lib/exq/mock.ex:160]: I can't think of any specific reason. What your use case? do you depend on the timestamp for enqueue_at or enqueue_in in your tests? I am open to fix this if there are use cases I didn't consider.

Comment (sergey-chechaev) [reply lib/exq/mock.ex:160]: Yes, we need to check is "enqueued_at" equal Date Time that we set

Comment (ananthakumaran) [reply lib/exq/mock.ex:160]: fixed by https://github.com/akira/exq/pull/449

**Pull request merged**